dependencies/populate-mgnifyr-cache.R

```diff
@@ -3,7 +3,8 @@ library(MGnifyR)
 
 mg <- mgnify_client(usecache = T, cache_dir = '/home/jovyan/.mgnify_cache')
 
-# For the "Comparative Metagenomics 1" notebook
+# For the "Comparative Metagenomics" notebook
 tara_all = mgnify_analyses_from_studies(mg, 'MGYS00002008')
 metadata = mgnify_get_analyses_metadata(mg, tara_all)
-
+clean_acc=c('MGYA00590519','MGYA00590489','MGYA00590465','MGYA00590530','MGYA00590487','MGYA00590476','MGYA00590516','MGYA00590561','MGYA00590536','MGYA00590567','MGYA00589017','MGYA00590491','MGYA00590493','MGYA00593140','MGYA00590557','MGYA00589029','MGYA00589036','MGYA00593111','MGYA00593018','MGYA00593225','MGYA00590531','MGYA00589062','MGYA00590459','MGYA00589052','MGYA00593131','MGYA00589048','MGYA00590527','MGYA00590576','MGYA00589058','MGYA00590558','MGYA00590550','MGYA00593113','MGYA00593015','MGYA00593121','MGYA00590572','MGYA00590545','MGYA00590518','MGYA00593126','MGYA00590526','MGYA00590500','MGYA00590570','MGYA00590520','MGYA00590443','MGYA00589013','MGYA00590449','MGYA00589021','MGYA00593130','MGYA00589047','MGYA00589042','MGYA00590577','MGYA00590470','MGYA00590473','MGYA00593216','MGYA00590562','MGYA00590464','MGYA00590484','MGYA00590462','MGYA00590565','MGYA00590439','MGYA00590472','MGYA00590566','MGYA00590552','MGYA00590485','MGYA00593133','MGYA00590544','MGYA00590455','MGYA00590437','MGYA00589044')
+ps = mgnify_get_analyses_phyloseq(mg, clean_acc)
```

Comment:
brilliant

Reply:
I'll update this again, as I change the data once more